### ISSUE TITLE
How to npm install with create-react-app

### ISSUE THREAD

**Issue by Noitidart** [ReactTraining/react-router-addons-controlled#9]
I am very new to using non-UMD packages. I am using create-react-app.

I am not able to install react-router-v4 or this.  I do as the quick start here says - https://react-router.now.sh/quick-start - which is - `npm install react-router history` but this seems to install react-router v3.

When I do `npm install --save react-router-addons-controlled` it seems to install a react-router within as a dependency. But I want to to use react-router-v4 latest from github so it has bug fix for this - https://github.com/ReactTraining/react-router/issues/4336#issuecomment-276038789

Also the example doesn't work right, it says here: https://github.com/ReactTraining/react-router-addons-controlled/blob/master/redux-example/index.js

That I should do:

     import Router from '../modules/ControlledBrowserRouter'

However that fails.

May you please help a newbie.

**Comment by Noitidart** [comment]
I think we need to change this `import StaticRouter from 'react-router/StaticRouter'` to `import { StaticRouter } from 'react-router'` in here - https://github.com/ReactTraining/react-router-addons-controlled/blob/master/modules/ControlledBrowserRouter.js#L3

**Comment by ingro** [comment]
I don't think that this could work with react-router@beta without some adjustments because the API have changed a lot...

**Comment by Noitidart** [comment]
@ingro I think you're right, I was modifying it a bit and wasn't having any luck with it. Thanks for your comment.

**Comment by ingro** [comment]
Me neither @Noitidart , replacing the old `StaticRouter` with the new wasn't enough. Let's hope that this library gets updated soon!
